lib/util.js

```diff
@@ -1057,3 +1057,81 @@ exports._exceptionWithHostPort = function(err,
 // process.versions needs a custom function as some values are lazy-evaluated.
 process.versions[exports.inspect.custom] =
   (depth) => exports.format(JSON.parse(JSON.stringify(process.versions)));
+
+const kCustomCallbackifiedSymbol = Symbol('util.callbackify.custom');
+const kCustomCallbackifiedToSplat = Symbol('util.callbackify.toSplat');
+const kCustomCallbackifiedProject = Symbol('util.callbackify.project');
```

Comment:
I don't think we need these properties - they complicate the PR and with destructuring syntax and a uniform convention none of the benefits it would bring to `util.promisify` apply here IMO.

For example, it is required for `fs.exists` to work in `promisify` but there is no equivalent here.